.ci/run.sh

```diff
@@ -5,6 +5,12 @@ if [ -d "${HOME}/.local/bin" ]; then
   export PATH="${HOME}/.local/bin:${PATH}"
 fi
 
+# Setup environment for using cached libraries/binaries
+export CI_DEPS_PATH="${HOME}/cideps"
+export PATH="${PATH}:${CI_DEPS_PATH}/bin"
+export PKG_CONFIG_PATH="${CI_DEPS_PATH}/lib/pkgconfig"
+export LD_LIBRARY_PATH="${CI_DEPS_PATH}/lib/"
```

Comment:
I would append to `PKG_CONFIG_PATH` and `LD_LIBRARY_PATH` over nuking whats there. This way if someone runs `run.sh` locally, they can set stuff to however they need it.

Reply:
Fixed